client/src/components/form/zodSchema/model.ts

```diff
@@ -0,0 +1,21 @@
+import * as z from "zod";
+
+const mailModel = z.string().email();
+
+const userNameModel = z
+  .string()
+  .min(3)
+  .max(128)
+  .regex(/^[\w_]+?$/, {
```

Comment:
`\w` は `[a-zA-Z0-9_]` です。
```suggestion
  .regex(/^\w+?$/, {
```

Reply:
[eb3467d](https://github.com/Growthers/enshita/pull/145/commits/eb3467dd3b637e252759b2c5355d3e0b332e317a)